### PR TITLE
[IMP] Domain names: improve current doc

### PR DESCRIPTION
**Pull request by Donapi**
taskid-3595179

### CODE REVIEW

**Review by chiaraprattico — CHANGES_REQUESTED** [review]

File: content/administration/maintain/domain_names.rst [hunk 1]
```diff
@@ -2,139 +2,141 @@
...
 
-.. note::
-   Please `submit a support ticket <https://www.odoo.com/help>`_ if you need further assistance to
-   manage your domain name.
+**Option 2**
+
+You can configure your MX records to use your email server or solution such as Google Workspace.
+Create the DNS records required to use your domain name with a third-party email service.
+To manage DNS records, go to your Odoo portal, click :guilabel:`My Databases`, then click the gear
+icon, :guilabel:`Domain names`, and :guilabel:`DNS`.
```
[the remark this answers is not in the document]
Comment (chiaraprattico) [reply content/administration/maintain/domain_names.rst:136]: You can configure your MX records to use your email server such as Google Workspace. To do so, create the required DNS records to use your domain name.
To manage DNS records, go to your Odoo portal, click :guilabel:`My Databases`, then click the gear
icon, :guilabel:`Domain names`, and :guilabel:`DNS`.

File: content/administration/maintain/domain_names.rst [hunk 2]
```diff
@@ -2,139 +2,141 @@
...
-==================
+Register a free domain name with Odoo
+=====================================
 
-Having a **good domain name** is as important to your branding as the name of your business or
-organization as it is the first thing your visitors will notice. We recommend you keep them *simple,
-short, easy to remember and spell*.
+When creating your site on Odoo, you can enjoy free domain registration for the first year. After
+this period, Odoo continues to manage your domain in partnership with **Gandi**, the domain name
+registrar, and you will be charged at the renewal rates as indicated on `their site (renewal column)
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/administration/maintain/domain_names.rst:39]: I replaced "enjoy" by "get" as discussed

File: content/administration/maintain/domain_names.rst [hunk 3]
```diff
@@ -2,139 +2,141 @@
...
 
-A **subdomain** is a domain that is a part of another domain. It often refers to the additional part
-that comes before the main domain name. Traditionally, most websites use the ``www.`` subdomain,
-but any string of letters can be used as well. You can use subdomains to direct your visitors to
-other websites than your main website or to specific pages (e.g., ``experience.odoo.com`` points to
-a specific page.)
+.. note::
+   - A one-year free domain name is also available for free Odoo Online databases (if you installed
+     one app only, for example). In this case, Odoo reviews your request and website to avoid abuse.
+     This process can take several days due to the success of the offer.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/administration/maintain/domain_names.rst:47]: Rephrased: This process can take several days due to the high number of requests.

File: content/administration/maintain/domain_names.rst [hunk 4]
```diff
@@ -364,3 +366,33 @@ In the :guilabel:`Domain` field, fill in the web address of your website (e.g.,
...
+using the :guilabel:`Console Google Search` available in your website's configuration.
+
+.. Important::
+   You cannot remove the subdomain dbname.odoo.com if Odoo hosts you. However, the subdomain
+   dbname.odoo.com exploration by search engines is blocked when you add a custom domain name in the
+   configuration of your website.
+
+.. seealso::
+   - `Google Search Console Help - Verify your site ownership <https://support.google.com/webmasters/answer/9008080>`_
+   - `Bing Webmaster Tools - Add and Verify site <https://www.bing.com/webmasters/help/add-and-verify-site-12184f8b>`_
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/administration/maintain/domain_names.rst:398]: you're right!

File: content/administration/maintain/domain_names.rst [hunk 5]
```diff
@@ -364,3 +366,29 @@ In the :guilabel:`Domain` field, fill in the web address of your website (e.g.,
 
 .. seealso::
   - :doc:`/applications/general/email_communication/email_servers`
+
+.. _domain-name/indexing:
+
+Indexing of domain names by search engines
+==========================================
```
Comment (ksc-odoo) [inline content/administration/maintain/domain_names.rst:373]:
```suggestion
Indexing domain names by search engines
=======================================
```

File: content/administration/maintain/domain_names.rst [hunk 6]
```diff
@@ -364,3 +366,29 @@ In the :guilabel:`Domain` field, fill in the web address of your website (e.g.,
...
+
+Search engines like Google and Bing rely on web crawlers (:dfn:`robots that explore and analyze
+the web`) to discover and index websites and their related domain names. These crawlers find new
+website links through existing pages, enabling search engines to automatically index domain names
+when their URLs are mentioned elsewhere online.
```
Comment (ksc-odoo) [inline content/administration/maintain/domain_names.rst:378]:
```suggestion
Search engines, like Google and Bing, rely on web crawlers (:dfn:`robots that explore and analyze
the web`) to discover and index websites, and their related domain names. These crawlers find new
website links through existing pages, enabling search engines to automatically index domain names
when their URLs are mentioned elsewhere online.
```

File: content/administration/maintain/domain_names.rst [hunk 7]
```diff
@@ -364,3 +366,29 @@ In the :guilabel:`Domain` field, fill in the web address of your website (e.g.,
...
+Improving the appearance and positioning of web pages on search engines is a practice called
+:ref:`Search Engine Optimization <pages/seo>`.
+
+Some search engines provide tools for web admins, such as
+`Google Search Console <https://search.google.com/search-console>`_ and
+`Bing Webmaster Tools <https://www.bing.com/webmasters>`_, to help you analyze and improve your
+page ranking. You must prove that you own your domain name to use these services. One way to verify
+the ownership of your domain name is by adding a DNS record (you can do this for :ref:`domain names
+registered with Odoo <domain-name/odoo-manage>` and for domain names managed by other providers) or
+using the :guilabel:`Console Google Search` available in your website's configuration.
```
Comment (ksc-odoo) [inline content/administration/maintain/domain_names.rst:389]:
```suggestion
Some search engines provide tools for web admins, such as:

`Google Search Console <https://search.google.com/search-console>`_ and `Bing Webmaster Tools
<https://www.bing.com/webmasters>`_, to help users analyze and improve page ranking.

Users **must** prove themselves as the owner of the domain name to use these services.

One way to verify the ownership of the domain name is by adding a DNS record (users can do this for
:ref:`domain names registered with Odoo <domain-name/odoo-manage>` and for domain names managed by
other providers) or by using the :guilabel:`Console Google Search` available in the website's
configuration.
```

File: content/administration/maintain/domain_names.rst [hunk 8]
```diff
@@ -364,3 +366,29 @@ In the :guilabel:`Domain` field, fill in the web address of your website (e.g.,
...
+
+.. Important::
+   You cannot remove the subdomain dbname.odoo.com if Odoo hosts you. However, the subdomain
+   dbname.odoo.com exploration by search engines is blocked when you add a custom domain name to the
+   configuration of your website.
```
Comment (ksc-odoo) [inline content/administration/maintain/domain_names.rst:394]:
```suggestion
   Users **cannot** remove the subdomain dbname.odoo.com if Odoo is the host. However, the subdomain
   dbname.odoo.com exploration by search engines is blocked when users add a custom domain name to
   the configuration of the website.
```

File: content/applications/websites/website/pages/seo.rst [hunk 9]
```diff
@@ -1,3 +1,5 @@
+.. _pages/seo:
+
 ================================
 Search Engine Optimisation (SEO)
```
Comment (ksc-odoo) [inline content/applications/websites/website/pages/seo.rst:4]:
```suggestion
Search Engine Optimization (SEO)
```

**Review by ksc-odoo — COMMENTED** [review]
@Donapi Just finished reviewing this doc. Once the comments I left have been addressed, this PR will be ready for the next stage.

**Comment by xpl-odoo** [comment]
> @Donapi Just finished reviewing this doc. Once the comments I left have been addressed, this PR will be ready for the next stage.

Hi @ksc-odoo ! Thanks for the review and valuable suggestions.
FYI, you were automatically pinged because @Donapi modified a file under the Email Marketing team's responsibility (`content/applications/general/email_communication/email_servers.rst`). In such cases, you should only review files under your responsibility 😄

**Comment by xpl-odoo** [comment]
@robodoo r+

**Pull request closed without merging**